.flake8

```diff
@@ -0,0 +1,9 @@
+[flake8]
+doctests = True
+ignore =
+    # W503 and W504 are mutually exclusive, so one or the other must be ignored.
+    # PEP 8 recommends line break before, so we keep W504.
+    W503
+max-complexity = 10
+max-line-length = 120
+select = E,W,F,C,N
```

Comment:
Add a new line here.

Reply:
I do have a workflow to check basic formatting styles:
https://github.com/per1234/.github/blob/development/workflow-templates/check-general-formatting.md

Reply:
Nice, thanks @per1234. You know what to do @umbynos. :wink:

Reply:
I have been a bit unsure about that workflow because not everyone is so uptight about formatting as I am. I definitely like it though. I have [a collection of `find` commands](https://github.com/per1234/formatting-checks/blob/master/.travis.yml#L14-L27) I added to all my CI configurations for years to accomplish essentially the same thing, but I think this approach of using the "editorconfig-checker" is a lot nicer.

I think it would be beneficial for the Tooling Team to review the available workflows and settle on a standard set to apply to a given project type (and maybe some like this one that can be used in every repo).

Reply:
Well it could be a nice opportunity to try it out! Thanks a lot @per1234. I've just added it